# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/bindings/moz.build
# Commit: d838dc7d2bb8
# Full Hash: d838dc7d2bb89ab5c9c2e8e93895db1c9310560f
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 4
# Description:
#   Bug 1654112 - Update include paths for moved upstream code. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D130059
# ==============================================================================

diff -r 587c40771588 -r d838dc7d2bb8 dom/bindings/moz.build
--- a/dom/bindings/moz.build	Fri Oct 23 13:49:51 2020 -0400
+++ b/dom/bindings/moz.build	Fri Oct 23 14:02:54 2020 -0400
@@ -99,7 +99,6 @@
     "/media/webrtc/",
     "/netwerk/base/",
     "/third_party/libwebrtc",
-    "/third_party/libwebrtc/webrtc",
 ]
 
 LOCAL_INCLUDES += ["/third_party/msgpack/include"]